docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3330,6 +3330,23 @@ function toggleBreakout() {
   }
 }
 
+async function toggleBrb() {
+  const meeting = getCurrentMeeting();
+
+  if (meeting) {
+    const enabled = document.getElementById('brb').checked;
+    try {
+      const result = await meeting.beRightBack(enabled);
+      console.log(`meeting.beRightBack(${enabled}): SUCCESS | result: ${result}`);
+    } catch (error) {
+      console.error(`meeting.beRightBack({${enabled}): ERROR`, error);
```

Comment:
I believe that uppercase messages can attract attention and may be useful. However, if everything in the console were written this way, I'm not sure it would be helpful or easy to read. I haven't seen any patterns for logs using this approach, but I like the idea of having concise messages. I have updated this part a little bit, if anything please let me know @SomeBody16 .